guides/common/modules/ref_ipv6-and-ipv4-requirements.adoc

```diff
@@ -0,0 +1,43 @@
+[id="ipv6-and-ipv4-requirements"]
+= IPv6 and IPv4 requirements
+
+You can install {Project} in an IPv4 network or in an IPv6 network.
+ifdef::satellite[]
+Dual-stack {Project} installation that uses both IPv4 and IPv6 is not supported.
+endif::[]
+
+The following requirements apply to installations in an IPv4 network:
+
+* Ensure an IPv6 loopback is configured on the base system.
+The loopback is typically configured by default.
+Do not disable it.
+* Do not disable IPv6 in kernel by adding the `ipv6.disable=1` kernel parameter.
+
+The following requirements apply to installations in an IPv6 network:
+
+ifdef::satellite[]
+* Deploy an external HTTP proxy server that supports both IPv4 and IPv6.
+This is required because Red Hat Content Delivery Network distributes content only over IPv4 networks, therefore you must use this HTTP proxy to pull content into the {Project} on your IPv6 network.
+You must configure {Project} to use this dual-stack (supporting both IPv4 and IPv6) HTTP proxy as the default HTTP proxy.
+endif::[]
+ifndef::satellite[]
+* If you rely on content from IPv4-only networks, use an external dual-stack HTTP proxy.
+Configure {Project} to use this dual-stack (supporting both IPv4 and IPv6) HTTP proxy as the default HTTP proxy.
+endif::[]
```

Comment:
This probably should not be rendered for foreman-el/foreman-deb, as there is no concept of "content" there.